### DISCUSSION TITLE
Having issue to apply other designs of TextField type in CoreForm

### DISCUSSION THREAD

**Discussion by PritamBag** [wrappid/.github#24] [General]
In CoreForm there is only one type of design TextField available which is the standard one similar to the mui library but I want to apply the other two as well just like the photo given below 👇

![Screenshot 2024-07-04 140522](https://github.com/wrappid/.github/assets/130136389/f3c67e96-2546-4fd8-80d9-d6f2a32126ad)

to get the complete documentation reference 👇
https://mui.com/material-ui/react-text-field/#basic-textfield

**Comment by samhere17** [comment]
Did you try passing the variant prop?

If you are using the json format to pass field metadata, try using a key "field-props". I am not sure if it will work though.
If it does not, we will have to fix the package to support this.
